spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/archaius/ArchaiusAutoConfiguration.java

```diff
@@ -165,14 +158,30 @@ protected void configureArchaius(ConfigurableEnvironmentConfiguration envConfig)
 			config.setContainerConfigurationIndex(config
 					.getIndexOfConfiguration(appOverrideConfig));
 
-			ConfigurationManager.install(config);
-		}
-		else {
+			addArchaiusConfiguration(config);
+		} else {
 			// TODO: reinstall ConfigurationManager
 			log.warn("Netflix ConfigurationManager has already been installed, unable to re-install");
 		}
 	}
 
+	private void addArchaiusConfiguration(ConcurrentCompositeConfiguration config) {
```

Comment:
Can you give me more details on this (or an example :-) ) ? This method just extended the functionality of the ConfigurationManager.install(..) that was previously called from method configureArchaius(...) line 168